modules/replicator/src/main/java/org/apache/ignite/internal/replicator/ReplicaManager.java

```diff
@@ -1028,4 +1087,357 @@ public Set<ReplicationGroupId> startedGroups() {
                 .map(Entry::getKey)
                 .collect(toSet());
     }
+
+    @TestOnly
+    public boolean isReplicaPrimaryOnly(ReplicationGroupId groupId) {
+        return replicaLifecycle.isReplicaPrimaryOnly(groupId);
+    }
+
+    private static class ReplicaLifecycle {
```

Comment:
The naming clashes with PartitionReplicaLifecycleManager. Any other ideas for the naming?

Reply:
what about ReplicaStateManager?